Нагаев Дмитрий

fng3r opened the pull request:
[Посмотреть решение](https://rawgit.com/fng3r/markup-task-4/master/index.html)

fng3r committed:
done

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bazhenova commented:
Привет. Хорошо. Сначала пройдемся по внешнему виду.
1. При отображении списком некоторые изображения телефонов большие, некоторые маленькие. Выглядит не очень
![image](https://user-images.githubusercontent.com/12410663/32225766-568ae0d8-be68-11e7-9f68-85afcfd5cc9f.png)

2. Не реализовано требование с эффектом при наведении на всю плитку в целом

3. У тебя все картинки одинаковой высоты. А если добавить картинку меньшего размера, будет ли такое же выравнивание по нижней линии фоточки?

4. Много пустого места
![image](https://user-images.githubusercontent.com/12410663/32225996-1ab153d4-be69-11e7-8204-33b59b571a9c.png)

5. Если ты делаешь дополнительное задание, то
>обратите внимание, что есть оценка в ползвезды

У тебя не смогла поставить в ползвезды

6. Когда выбраны все звезды, то при наведение на предыдущие хочется, чтобы показывалось, сколько выберется, если нажать на кнопку

7. (Лично моё, можешь не править) Смартфоны. Но.. ведь.. котики.. 😞

bazhenova commented:
🍅

fng3r commented:
@bazhenova при наведении на плитку увеличивается описание

bazhenova commented on index.html:
```diff
@@ -1,9 +1,324 @@
+                title="Samsung Galaxy A5, White"
+                alt="Samsung Galaxy A5, White">
+            <a href="#" class="item-name" 
+               title="Samsung SM-A510F Galaxy A5, Whiteeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee">
```
Копипаста в `title` с самим содержанием. Не очень как-то

bazhenova commented on index.css:
```diff
@@ -0,0 +1,112 @@
+
+.item-name:hover
+{
+    color: red;
```
Цвет в hex

bazhenova commented on index.css:
```diff
@@ -0,0 +1,112 @@
+{
+    direction: rtl;
+    text-align: left;
+    user-select: none;
```
Правило без префиксов нужно указывать после правил с префиксами

bazhenova commented on index.css:
```diff
@@ -0,0 +1,112 @@
+
+.item-description
+{
+    display: table;
```
Для чего это?

bazhenova reviewed (commented):
При наведении на плитку у меня не увеличивается описание. Еще несколько замечаний вдогонку. 

Если все-таки собрался сделать доп. задание, то добавь картинку другого размера, чтобы было заметно выравнивание по нижней линии. И не забудь сделать отметку в ползвезды

🍅

fng3r committed:
+ reduced distance between items
+ images in list mode now have same sizes
+ selected rating becomes brighter while hovering

fng3r committed:
set item name same as the title, some code review fixes

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

fng3r commented:
> При наведении на плитку у меня не увеличивается описание.

:thinking: 
```
.item:hover .item-description
{
    font: 16px 'Bookman Old Style', serif;
}
```

fng3r committed:
set description font-size relatively to item font-size

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

fng3r committed:
made half-star rating

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

fng3r committed:
fixed bug with star color

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bazhenova commented:
Принято
